delivery_unifaun_base/models/delivery.py

```diff
@@ -1234,6 +1247,7 @@ def get_order_vals_from_pickings(self, pickings):
             'company_id': picking.company_id.id,
             'date': max([p.date_done for p in pickings]),
             'package_ids': self.get_package_values_from_pickings(pickings),
+            #'barcode_package_ids': picking.barcode_package_ids
```

Comment:
Remove or keep this commented code?
Add a comment on why we should keep it as commented code